docker/skopeo_runner/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM fedora:33
+
+# TODO: should the image contain any special labels?
```

Comment:
It's a metadata about the image. So, we could start with adding maintainer & description labels. 

LABEL Maintainer=" " \
           Description=" "